### PR TITLE
Support Rails 6.1

### PR DESCRIPTION
**Pull request by santib**
- [x] Wait until Rails 6.1 is released

### CODE REVIEW

**Comment by thesimpledev** [comment]
@santib Any updates here? Having issues with Rails 6

**Comment by santib** [comment]
> @santib Any updates here? Having issues with Rails 6

Hi @thesimpledev, Actually the versions `1.0.0` and `1.1.0` of this gem do already work with Rails 6 😄 You can see a working sample app here https://github.com/Ricoch/Issue55Example

This PR is to add support to Rails 6.1 which hasn't yet been released https://rubygems.org/gems/rails/versions

**Review by jpascual1994 — APPROVED** [review]
LGTM 🚀

File: active_storage_base64.gemspec [hunk 1]
```diff
@@ -12,10 +12,10 @@ Gem::Specification.new do |s|
   s.homepage = 'https://github.com/rootstrap/active-storage-base64'
   s.email = 'ricardo@rootstrap.com'
 
-  s.required_ruby_version = '>= 2.2.2'
+  s.required_ruby_version = '>= 2.5.0'
```
Comment (santib) [inline active_storage_base64.gemspec:15]: this was just to match Rails required ruby version

**Review by martinjaimem — APPROVED** [review]

**Review by juanmanuelramallo — APPROVED** [review]

**Pull request merged**